### PR TITLE
Create increase-limit-on-existing-paid-plan.md

### PR DESCRIPTION
**Pull request by mitchwh24**
new FAQ to try and stop limit requests through the support channel

### CODE REVIEW

File: articles/faq/increase-limit-on-existing-paid-plan.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+ title: "If i stay on my plan can i increase my daily limits?"
```
Comment (ewandennis) [inline articles/faq/increase-limit-on-existing-paid-plan.md]: `If I stay on my plan, can I increase my daily limits?`

File: articles/faq/increase-limit-on-existing-paid-plan.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+ title: "If I stay on my plan, can I increase my daily limits?"
+ description: "Your plan settings will remain as they are now. If you need a larger daily limit, please upgrade to one of our new plans. Please note that daily limits are always based on your account’s sending performance and history with SparkPost and are reviewed on a case by case basis."
+ ---
```
Comment (richleland) [inline articles/faq/increase-limit-on-existing-paid-plan.md]: add line break between the meta and the actual content

File: articles/faq/increase-limit-on-existing-paid-plan.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+ title: "If I stay on my plan, can I increase my daily limits?"
```
Comment (richleland) [inline articles/faq/increase-limit-on-existing-paid-plan.md]: Need to remove space in front of `title` and `description` - this metadata is YAML and the syntax is strict. If in doubt, you should always click the "view" button in github to see if things parse properly.

File: articles/faq/increase-limit-on-existing-paid-plan.md (hunk 2 above)
Comment (mitchwh24) [reply articles/faq/increase-limit-on-existing-paid-plan.md]: done

**Pull request merged**